docs/online/src/faq/host_install_addins.rst

```diff
@@ -0,0 +1,55 @@
+What are WOPI host preinstall add-ins, what are tips to use them?
+==================================================================================================
+
+When a document is opened or created from a WOPI host, the WOPI host can provide a list of Office Web add-ins to
+be used with the document. The list of add-ins are provided via as Form field values as part of POST request sent
```

Comment:
`via as Form field` seems like a typo.

Reply:
you probably be right, let me check the official name.